Model/Sync/Catalog/Processor.php

```diff
@@ -341,13 +341,15 @@ public function syncItems($collectionItems, $storeId, $visibleVariants = false)
                 if (!$visibleVariants) {
                     $parentData = $this->pushParentData((int)$itemId, $tempSqlArray, $parentData, $parentIds);
                 }
-                $syncDataSql = [];
-                $syncDataSql[] = $tempSqlArray;
-                $this->insertOnDuplicate(
-                    'yotpo_product_sync',
-                    $syncDataSql
-                );
-                $sqlData[] = $tempSqlArray;
+                if ($tempSqlArray) {
```

Comment:
@manaftryzens see [discussion](https://github.com/YotpoLtd/magento2-module-core/pull/107/files#r810936268) on the [original PR](https://github.com/YotpoLtd/magento2-module-core/pull/107).